src/tests/index.test.js

```diff
@@ -33,6 +42,46 @@ describe('serverless-plugin-cloudfront-distribution-tags', function() {
 
    });
 
+   describe('_modifyTemplate', function() {
```

Comment:
This should have a test for the case where an exclude list (or our plugin's config object) is not supplied.